dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/JSONTool.java

```diff
@@ -247,13 +249,35 @@ public Object generate(Object o) {
     return new JSONObject(o);
   }
 
+  private final static Class LIST_MAP_CLASS = new ArrayList<Map<String, Object>>().getClass();
+  private final static Class MAP_CLASS      = new HashMap<String, Object>().getClass();
```

Comment:
Codacy found an issue: [Avoid instantiating an object just to call getClass() on it; use the .class public member instead](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6315721)